tests/tests/neg/constructors/test00.rs

```diff
@@ -0,0 +1,14 @@
+#[flux::refined_by(x: int, y: int)]
+pub struct X {
+    #[flux::field(u32[x])]
+    x: u32,
+    #[flux::field(u32[y])]
+    y: u32,
+}
+
+#[flux::sig(fn (x: X[@old_x]) -> X[Y { y: 2, x: 1 }])] //~ ERROR cannot find value `Y` in this scope
```

Comment:
We typically put tests related to error messages that are not refinement-type errors in `neg/error_messages` inside the corresponding directory of the stage (wf in this case)

Reply:
Ok sounds good! I just put the tests in the right places (I think)